packages/runtime/src/state/react-builder-preview.ts

```diff
@@ -199,12 +200,20 @@ function getElementSize(element: HTMLElement): Size {
   }
 }
 
-function lockDocumentScroll(): () => void {
-  const lastDocumentOverflow = window.document.documentElement.style.overflow
-  window.document.documentElement.style.overflow = 'hidden'
+function lockDocumentScroll(): ThunkAction<() => void, State, unknown, Action> {
+  return dispatch => {
+    const lastDocumentOverflow = window.document.documentElement.style.overflow
+    window.document.documentElement.style.overflow = 'hidden'
 
-  return () => {
-    window.document.documentElement.style.overflow = lastDocumentOverflow
+    const lastDocumentOnWheel = window.document.documentElement.onwheel
+    window.document.documentElement.onwheel = ({ deltaX, deltaY }) => {
```

Comment:
Why aren't we using `addEventListener` here?

Reply:
Is the reason we're not using the `scroll` event here that we've locked scrolling? Should we unlock scrolling instead  when in content mode? Perhaps this is something to worry about later.

Reply:
In Leo we're handling it using onWheel event, I'm using just using the same event:

<img width="396" alt="image" src="https://user-images.githubusercontent.com/13066728/170726024-e64ac2cf-4bdf-4b83-b9e5-abf019296dd4.png">

For why we're not using `addEventListener`, yea we could listen to `wheel` listener. I tried using `onwheel` instead and got error.

Even if we unlock the scroll, I think we still can't scroll left in content mode, because it's handled in Leo instead of scrolling the host.

Reply:
Cool just fixed that. Thanks for the review!